tools/packaging/scripts/lib.sh

```diff
@@ -112,4 +117,34 @@ get_config_version() {
 	else
 		die "failed to find ${config_version_file}"
 	fi
-}
\ No newline at end of file
+}
+
+# Gets a list of files and/or directories to build a SHA-256 from their contents.
+# Returns the SHA-256 hash if succeeded, otherwise an empty string.
+sha256sum_from_files() {
```

Comment:
Are you going to delete that method on `tests/.ci/lib.sh`?

Reply:
@wainersm yes that is a possibility

Reply:
@wainersm I used the function from the test repository and remove it from here